scripts/prettier/index.js

```diff
@@ -52,11 +48,17 @@ const config = {
 
 function exec(command, args) {
   console.log('> ' + [command].concat(args).join(' '));
-  var options = {};
-  return execFileSync(command, args, options).toString();
+  var options = {
+    cwd: process.cwd(),
+    env: process.env,
+    stdio: 'pipe',
+    encoding: 'utf-8',
+  };
+  const childProcess = spawnSync(command, args, options);
+  return childProcess.stdout.trim();
```

Comment:
Sorry, I actually meant more specifically- in this file- it didn't seem like we actually wanted/needed untrimmed output. I agree with you in general.

But I don't feel strongly about this so I'll revert that part of the change.